src/utils/aotf.js

```diff
@@ -905,13 +905,13 @@ async function _mutateError (mutationName, message, response) {
   // open a user alert
   await store.dispatch(
     'setAlert',
-    new Alert(`Command failed: ${mutationName} - ${message}`, 'error')
+    new Alert(err, 'error', `Command failed: ${mutationName} - ${err}`)
```

Comment:
This was swallowing the stack trace meaning I couldn't investigate Chris Bunney's ApolloError